docs/src/DevDocs/AcceptableUnicode.md

```diff
@@ -1,12 +1,11 @@
 # Acceptable Unicode characters
```

Comment:
Maybe it would be better to have a list of forbidden unicode characters? The list should be smaller and you can easily add the justification right afterwards.

Reply:
Just spoke with Kiran about this, it might actually be a far longer list than acceptable unicode characters.

Reply:
enable superscript and subscripts please then :) we are doing it anyways